crates/hotshot/src/traits/networking/libp2p_network.rs

```diff
@@ -149,11 +150,11 @@ struct Libp2pNetworkInner<T: NodeType> {
     /// handle to control the network
     handle: Arc<NetworkNodeHandle<T>>,
     /// Message Receiver
-    receiver: UnboundedReceiver<Vec<u8>>,
```

Comment:
If this is locked because we use multiple receivers can we use async_brodcast instead